app/Services/MapboxGeocodingService.php

```diff
@@ -26,4 +29,35 @@ public function getCoordinatesFromApi(string $cityName, string $countryName): ar
             return [];
         }
     }
+
+    public function getPlaceFromApi(string $lat, string $long): array
+    {
+        $token = env("MAPBOX_TOKEN");
+        $client = new Client();
```

Comment:
It would be good to move Client to the constructor. With this `new` keyword it will be really hard to test it.

Reply:
now it is in  function's constructor

Reply:
I mean to move Client to the __constructor(). 
Then this service class will need to be injected too in constructors.
This will allow you to write tests for this MapboxGeocoding service.

Reply:
But maybe it will be better to do it in another task, cause this change will affect all importers.
Then you will be able to write tests for mapbox service.

Reply:
Created an issue.
https://github.com/blumilksoftware/escooters/issues/82
It would be great to do it as soon as possible.

Reply:
I'll try!